src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GenerateVCFFromPosteriors.java

```diff
@@ -0,0 +1,224 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.variantcontext.writer.VariantContextWriter;
+import org.apache.commons.lang3.tuple.ImmutableTriple;
+import org.apache.commons.lang3.tuple.Triple;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.ChunkedCopyNumberPosteriorCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.SimpleIntervalCollection;
+import org.broadinstitute.hellbender.utils.text.XReadLines;
+import org.broadinstitute.hellbender.utils.tsv.TableUtils;
+import org.broadinstitute.hellbender.utils.variant.GATKVariantContextUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Create a VCF given the output of gCNV caller
+ *
+ * <p> This tool takes a list of directories of chunked gCNV call directories containing posterior records for
+ * different copy number states and outputs a VCF. Note that the list should be sorted according order of the intervals
+ * in corresponding chunks. The VCF will contain an ALT allele for every non-reference copy
+ * number state specified in the posterior files.
```

Comment:
Since we assume REF = 2 for now, we should document that here.

Reply:
Done